### PR TITLE
ci: remove lll + fix other linters issues

### PR DESCRIPTION
**Pull request by hbollon**
I had to delete _isASCII_ and _(db *Database) listFieldWithCount_ functions which were unused.
Hope it's all good

### CODE REVIEW

**Review by cryptobioz — COMMENTED** [review]
Pretty good! Only few remarks regarding the way we should format long lines but it looks like there is no real best practices so it's up to us to choose what we prefer.

File: db/db.go [hunk 1]
```diff
@@ -32,14 +32,16 @@ var pageSize = 20
 // Init setups up the Database and a pointer to it
 func Init(config config.DBConfig, debug bool) *Database {
 	var err error
-	connString := fmt.Sprintf("host=%s port=%d user=%s dbname=%s sslmode=%s password=%s", config.Host, config.Port, config.User, config.Name, config.SSLMode, config.Password)
+	connString := fmt.Sprintf("host=%s port=%d user=%s dbname=%s sslmode=%s password=%s",
```
[the remark this answers is not in the document]
Comment (mcanevet) [reply db/db.go:35]: In the previous comment you suggested to add a line break just after the parenthesis but not here. I think we should have a consistent convention. I prefer having a line break after the parenthesis.

[the remark this answers is not in the document]
Comment (hbollon) [reply db/db.go:35]: I agree with @mcanevet and, moreover, it seems to me that this is the Go best practice for long function calls.

[the remark this answers is not in the document]
Comment (cryptobioz) [reply db/db.go:35]: I'm good with that, let's do it for all long function calls :+1:

**Comment by hbollon** [comment]
It's changed ! :smile:

**Review by cryptobioz — APPROVED** [review]

**Pull request merged**